recipes/django-axes/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "django-axes" %}
+{% set version = "5.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c8f1234003536e8eadc624745a2d302f9251943845a8df770c73e2bdb8782632
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - django
+    - black
+    - mypy
+    - prospector
+
+test:
+  imports:
+    - axes
```

Comment:
Are there more import tests that should be added? Right now the recipe passes even though packages listed in `install_requires` aren't actually being installed for the test.

Reply:
I'm not sure the best way to test this. We are just trying to add it to conda forge so we can add one of our packages

Reply:
That's fine.